src/index/ivf/ivf.cc

```diff
@@ -664,7 +664,11 @@ IvfIndexNode<T>::Serialize(BinarySet& binset) const {
         if constexpr (std::is_same<T, faiss::IndexBinaryIVF>::value) {
             faiss::write_index_binary(index_.get(), &writer);
         } else {
-            faiss::write_index(index_.get(), &writer);
+            if (versoin_ <= Version::GetMinimalSupport()) {
+                faiss::write_index_nm(index_.get(), &writer);
+            } else {
```

Comment:
pls check, when version = `knowhere-v0`, index could not be loaded.

Reply:
seems io_flags not set

Reply:
when "version == knowhere-v0", the index file will be loaded by previous version querynode